config/routes.rb

```diff
@@ -31,3 +33,4 @@
   match '/500', to: 'errors#internal_server_error', via: :all
   match '*path', to: 'errors#not_found', via: :all
 end
+# rubocop:enable Metrics/BlockLength
```

Comment:
Just wondering is there something else we can do instead

Reply:
I think using the rails config might be a better solution: https://github.com/toshimaru/rubocop-rails_config

Reply:
Probably best to do that in a separate PR and add it to all of our apps + boilerplate as well though.

Reply:
Just tested this, it doesn't work so great with the govuk rules.